.github/workflows/ci.yml

```diff
@@ -224,6 +224,28 @@ jobs:
       - name: Test
         run: yarn workspace @balancer-labs/v2-governance-scripts test
 
+  check-artifacts:
```

Comment:
Maybe rename these to 'check-deployment-artifacts' and 'check-deployment-addresses'

Reply:
Done, I don't really want to change the hardhat tasks though as it'll be unwieldy on the CLI.

Reply:
That's fine - the deployments context is already there since those belong to that package